tests/backup_helper.go

```diff
@@ -8540,3 +8542,39 @@ func GetUpdatedKubeVirtVMSpecForBackup(scheduledAppContextsToBackup []*scheduler
 	}
 	return nil
 }
+
+// GetPVCList will get the list of PVC's and returns the PVC names
+func GetPVCList() ([]string, error) {
+    pxBackupNamespace, err := backup.GetPxBackupNamespace()
+    if err != nil {
+        return nil, err
+    }
+
+    k8sCore := core.Instance()
+    pvcList, err := k8sCore.GetPersistentVolumeClaims(pxBackupNamespace, make(map[string]string))
+    if err != nil {
+        return nil, fmt.Errorf("failed to get PVCs in namespace %s: %w", pxBackupNamespace, err)
+    }
+
+    var pvcNameList []string
+    for _, pvc := range pvcList.Items {
+        pvcNameList = append(pvcNameList, pvc.Name)
+    }
+    return pvcNameList, nil
+}
+
+// ValidatePVCCleanup will compare the PVC list before and after the run
+func ValidatePVCCleanup(pvcListBefore, pvcListAfter []string) error {
+	// Check if the total number of the PVC's are the same
+	if len(pvcListBefore) != len(pvcListAfter) {
+		return fmt.Errorf("mismatch in pvc count")
```

Comment:
If there is a mismatch I would like to see the difference between the original PVC list and the current one

Reply:
@mkoppal-px I have done the requested changes, please review.